### DISCUSSION TITLE
VS Code startup setting not working

### DISCUSSION THREAD

**Discussion by SwabianCoder** [community/community#29178] [Codespaces]
In my devcontainer.json file in settings section I added `"workbench.startupEditor": "none"` setting. However I noticed the newly created codespace is ignoring the override of the setting. Also tested with value `"welcome"` which is being ignored as well.

**Comment by jaclynj** [comment]
Hi @SwabianCoder  - `workbench.startupEditor` is not a valid attribute for the `devcontainer.json` file for setting up codespaces. You can find the devcontainer docs with all the options here: https://code.visualstudio.com/docs/remote/devcontainerjson-reference

It looks like you can add the `workbench.startupEditor` setting in `defaultSettings.json` - docs here: https://code.visualstudio.com/docs/getstarted/settings#_default-settings

**Reply by SwabianCoder** [reply]
Hi @jaclynj 

as far as I can see you can customize VS Code's settings via devcontainer.json in a way I already do that. After the Codespace was built I can see the setting I set in JSON to be set:
<img width="1205" alt="image" src="https://user-images.githubusercontent.com/43047586/184298700-e7d53a7f-c01d-43e2-baf2-bd246b47a242.png">

But it's not taking effect.
[Here](https://github.com/SwabianCoder/ngx-toolset/blob/main/.devcontainer/devcontainer.json) you can find my devcontainer.json.
There you can see that I'm also overriding the VS Code setting `"workbench.colorTheme"` and that works.
